Dockerfile

```diff
@@ -17,14 +17,15 @@ RUN poetry config virtualenvs.in-project true
 # Setup project
 COPY pyproject.toml poetry.lock README.md ./
 COPY acapy_wallet_upgrade/ acapy_wallet_upgrade/
+COPY askar_tools/ askar_tools/
 COPY tests/ tests/
 RUN poetry build
 
 
 FROM --platform=linux/amd64 python:${python_version}-slim AS askar-upgrade
-COPY --from=base /usr/src/app/dist/acapy_wallet_upgrade-*-py3-none-any.whl /tmp/.
+COPY --from=base /usr/src/app/dist/askar_tools-*-py3-none-any.whl /tmp/.
```

Comment:
How do we make directories coexist here? This seems to be still adding the wallet upgrade code, but only installing the askar tools?

Reply:
I need to look at this again. I can't remember what I was doing but it is possible to do copy both directories and have them coexist. The scripts are defined in the root pyproject.toml file. The project itself needs to define one of the directories as the root project but the others will work. I'll try to explain this better when I get the effort closer to maturity.

Reply:
Ya this still copies all the tools but the dist will be called askar-tools now.